src/03-dice-roller/DiceRoller/DiceRoller/Program.cs

```diff
@@ -10,35 +10,37 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("> /roll 3d6 2d8");
+            // ask user input and make string array
+            Console.WriteLine("Insert command and dices to roll.");
             Console.WriteLine();
+            Console.Write(">");
+            string[] inputArray = Console.ReadLine().Split(null);
```

Comment:
Why are you doing .Split(null) when it should be char[] separator? If you want to split on space, do .Split(' ') instead.

Reply:
I took this `.Split(null)` example from the C# docs- if the separator parameter is null or contains no characters, white-space characters are assumed to be the delimiters.